packages/types/src/ethereum/interfaces.ts

```diff
@@ -22,7 +22,7 @@ export interface EthereumTransactionFilter {
    * @example
    * to: '0x220866B1A2219f40e72f5c628B65D54268cA3A9D',
    **/
-  to?: string;
+  to?: string | null;
```

Comment:
I don't think this needs to change. Code in the node can be updated instead
```suggestion
  to?: string;
```

Reply:
I think this still needs to be only optional string. The dictionary code should change if its wrong

Reply:
image I see that the test case includes this usage scenario. Shouldn’t we support assigning null?
![image](https://github.com/user-attachments/assets/e2d1f4a2-2afd-496e-b118-c24717725d3a)